lib/detection_rules.yml

```diff
@@ -1071,3 +1071,14 @@
         platform: Android
       - userAgent: "ARD Audiothek iOS Version 2.3.4, Build 41"
         platform: iOS
+
+- app: Samsung Podcast
+  match:
+    startsWith: "sp-agent"
+    includes: "sp-agent"
```

Comment:
Do we need both? I would keep the `startsWith` for now, as everything we see is the given string.

If we want to use `includes`, I would specify some additional word boundaries `\b` if possible, so that some weird user agent string is not mapped to Samsung Podcasts (even though I think the chances are low).

Reply:
Thanks for checking. Changed it and kept `startsWith` only. I can do a squash and force-push to clean up this PR afterwards.

Reply:
I think you removed `startsWith` :-)

Reply:
I'm sorry, you are right 🙁 Too warm in here... Fixed, squashed and force-pushed to clean up this mess.